core/src/main/java/hudson/model/DirectoryBrowserSupport.java

```diff
@@ -446,6 +452,17 @@ public long getSize() {
             return size;
         }
 
+        public long getLastModified() {
+            return lastModified;
+        }
+
+        public java.util.Calendar getLastModifiedAsCalendar() {
```

Comment:
Should probably be `@Restricted` when the only reason this exists is the Jelly view.

Reply:
ok, I will also add the javadoc and the `@since TODO`